pkg/core/mempool/mem_pool.go

```diff
@@ -108,6 +114,29 @@ func (mp *Pool) containsKey(hash util.Uint256) bool {
 	return false
 }
 
+// HasConflicts returns true if transaction is already in pool or in the Conflicts attributes
+// of pooled transactions or has Conflicts attributes for pooled transactions.
+func (mp *Pool) HasConflicts(t *transaction.Transaction, fee Feer) bool {
```

Comment:
Why `fee` here?

Reply:
We need `P2PSigExtensionsEnabled` here along with standard `containsKey` check. Should I make a separate method for conflicts attributes check?

Reply:
Oh, true, `P2PSigExtensionsEnabled` justifies that and one method is more than enough.